.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+**Note: for support questions, please use Discord**. This repository's issues are reserved for feature requests and bug reports.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-file-name-irregular-characters] Do not use `_` in a file name](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5470889)